go.sum

```diff
@@ -1,15 +1,70 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
There are a lot of additions to go.sum that look unrelated. Do they go away if you `go mod tidy` or are they coming in from the go-pdf/fpdf?

Reply:
no, these are coming from a new version of `golang.org/x/exp` (that was collected when this CL was rebased off on a attempt at upgrading Gio).
that new version of `x/exp` has a lot of new OpenTelemetry stuff in it.

I removed that new version of `x/exp` from the `vg/vgpdf` PR.